repair/repair.cc

```diff
@@ -1908,9 +1923,15 @@ future<> repair_service::replace_with_repair(locator::token_metadata_ptr tmptr,
     co_return co_await do_rebuild_replace_with_repair(std::move(cloned_tmptr), std::move(op), std::move(source_dc), reason);
 }
 
+future<> repair_service::init_metrics() {
```

Comment:
nitpick: this function didn't need to return a future at all. But I guess there's little harm that it does.

Reply:
It makes it more consistent with other init functions. 

```
--- a/repair/row_level.cc
+++ b/repair/row_level.cc
@@ -3012,6 +3012,7 @@ repair_service::repair_service(distributed<gms::gossiper>& gossiper,
 
 future<> repair_service::start() {
     return when_all_succeed(
+            init_metrics(),
             init_ms_handlers(),
             init_row_level_ms_handlers()
     ).discard_result();
```